llama-index-integrations/vector_stores/llama-index-vector-stores-opensearch/llama_index/vector_stores/opensearch/base.py

```diff
@@ -389,6 +392,41 @@ async def delete_by_doc_id(self, doc_id: str) -> None:
         }
         await self._os_client.delete_by_query(index=self._index, body=search_query)
 
+    async def delete_nodes(
```

Comment:
we should implement `async def delete_nodes()` and `async def aclear()` here -- the sync versions `delete_nodes()` and `clear()` can call the async version like

```python
asyncio.get_event_loop().run_until_complete(
    self.adelete_nodes(....)
)
```

Reply:
or, it looks like this got implemented twice? A little confused 😅 Maybe double check the method names and make sure a method is only implemented once, then it will be all good

Reply:
omg I'm silly, this is the client! All good!